README.md

```diff
@@ -89,7 +89,19 @@ bind mounts and exposing ports for you.
     
 Note that `<PROJECT_ROOT>` must be absolute.
 
-You will then be able to view the application on http://localhost:5000/    
+You will then be able to view the application on http://localhost:5000/
+
+### Testing (in docker)
+The `Dockerfile` has a special stage for running tests.
+
+To build:
+    
+     docker build --target test --tag <image>:<tag> .
+To run:
+
+    docker run --mount type=bind,source=<PATH_ON_LOCAL>,target=/app <image>:<tag> .
```

Comment:
docker run doesn't require the build context, so we don't need this final `.` (currently it's just being passed as an argument to pytest, which will assume the current directory by default anyway)